ConsistencyManager/ConsistencyManager.swift

```diff
@@ -95,6 +95,12 @@ open class ConsistencyManager {
     var listeners = [String: WeakListenerArray]()
 
     /**
+     This is an array of all the model update listeners.
+     These will get notified whenever anything changes in the Consistency Manager.
+     */
```

Comment:
Can you add documentation that explains why someone would want to have this kind of listener